static/elements/chromedash-callout.js

```diff
@@ -0,0 +1,136 @@
+import {LitElement, css, html} from 'lit-element';
+
+class ChromedashCallout extends LitElement {
+  static get properties() {
+    return {
+      signedin: {type: Boolean},
+      targetid: {type: String},
+      // TODO(jrobbins): Support sides other than "south".
+      side: {type: String}, // "north", "south", "east", or "west"
+      hidden: {type: Boolean},
+      top: {type: Number},
+      left: {type: Number},
+      cue: {type: String}, // String to send to the server when dismissed.
+      dismissedcues: {attribute: false},
+    };
+  }
+
+  constructor() {
+    super();
+    this.side = 'south';
+    this.hidden = true;
+    this.top = 0;
+    this.left = 0;
+    this.signedin = false;
+    this.dismissedcues = [];
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    if (this.dismissedcues.includes(this.cue)) {
+      // Don't show the cue because the user has already dismissed it.
+      return;
+    }
+    let targetEl = this.parentNode.querySelector('#' + this.targetid);
+    if (targetEl) {
+      this.attach(targetEl);
+    } else {
+      console.log('Callout target not found: #' + this.targetid);
+    }
+  }
+
+  static get styles() {
+    return css`
+      :host {
+        /* TODO(jrobins): Put this into a global theme.css file. */
+        --blue-900: #174EA6;
+
+        --callout-bg-color: var(--blue-900);
+        --callout-text-color: white;
+      }
+      #bubble {
+        position: absolute;
+        display: inline-block;
+        background: var(--callout-bg-color);
+        border-radius: 8px;
+        padding: 8px;
+        max-width: 24em;
+        color: var(--callout-text-color);
+        font-weight: bold;
+      }
+      #bubble[hidden] {
+        display: none;
+      }
+      #bubble:after {
+        content: '';
+        position: absolute;
+        border: 20px solid transparent;
+      }
+      /* Bubble will be located on the south side of target element. */
+      #bubble.south:after {
+        top: 0;
+        left: 40px;
+        width: 0;
+        height: 0;
+        border-bottom-color: var(--callout-bg-color);
+        border-top: 0;
+        margin-left: -20px;
```

Comment:
I think margins of this absolute positioned element can be achieved by updating `left: 20px; top: -20px` above.

Reply:
That is simpler.  Thanks!